add-on/chrome/bootstrap.js

```diff
@@ -485,6 +488,9 @@ var WindowListener = {
           // metadata about the page is available when this event fires.
           gBrowser.addEventListener("DOMTitleChanged", this);
           this._browserSharePaused = false;
+
+          this._mousemoveHandler = this.handleMousemove.bind(this);
```

Comment:
I think we can avoid `_mousemoveHandler` and just assign `handleMousemove` to the event listener. 
What's the use of keeping the second variable?